### DISCUSSION TITLE
Embedding The 3.0 Editor

### DISCUSSION THREAD

**Discussion by samuraininja360** [ScratchAddons/ScratchAddons#674] [Ideas] [answered]
So is it possible to get an `<iframe>` from the scratch editor because it is possible in MakeCode for Microbit. This also allows people to see your intricate code which shows how advanced you are anyway. Currently there is only an embed option for the project player. It actually looks bad but having the editor embedded would be handy.

**Comment by mxmou** [comment, marked as answer]
Yes, you could load `https://llk.github.io/scratch-gui/develop/#PROJECT ID` in an iframe.

**Comment by marcosilvams709-lab** [comment]
Olá, quero incorporar o scratch em meu google site não estou conseguindo, quando abre está incompleto.
